livestyled/models/device_form_data.py

```diff
@@ -0,0 +1,76 @@
+from livestyled.models.device import Device
+from livestyled.models.form import Form
+
+
+class DeviceFormData:
+    def __init__(
+            self,
+            id,
+            device_id,
+            form_id,
+            data,
+            created_at=None,
+            updated_at=None
+    ):
+        self._id = id
+        self.data = data
+        self.created_at = created_at
+        self.updated_at = updated_at
+        self._device = Device.placeholder(id=device_id)
+        self._form = Form.placeholder(id=form_id)
+
+    @classmethod
+    def create_new(
+            cls,
+            device: Device or str or int,
+            form: Form or str or int,
+            data: list,
+    ):
+        device_form_data = DeviceFormData(
+            id=None,
+            device_id=None,
+            form_id=None,
+            data=data,
+            created_at=None,
+            updated_at=None,
+        )
+        if isinstance(device, (str, int)):
+            device = Device.placeholder(id=device)
+        device_form_data._device = device
```

Comment:
should we just assign this right away within? 
`device_form_data._device = Device.placeholder(id=device)`

same with line 41 forms.

Reply:
but what if device is an instance of the device model? We wouldn't want to use the placeholder method, we would just assign it to the property.